HelperViews/ContextMenuActions.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MAUIDesigner.HelperViews
+{
+    internal static class ContextMenuActions
+    {
+        public static void DetachFromParent_Clicked(View targetElement, ContextMenu contextMenu, EventArgs e)
+        {
+            if (targetElement?.Parent is Layout parentLayout)
+            {
+                // Remove the target element from its parent
+                parentLayout.Children.Remove(targetElement);
```

Comment:
This will render it unusable since it's not part of the `designerFrame` absolute layout. Please update the parent to that.